sahara/manifests/init.pp

```diff
@@ -156,6 +156,7 @@
     ensure => $package_ensure,
     name   => 'openstack-sahara-ui',
     tag    => 'openstack',
+    notify  => Service['httpd'], # only restarts if change
```

Comment:
I'll be nitpicky and point out you need to move all the `=>` here for proper style.